### ISSUE TITLE
Long polling

### ISSUE THREAD

**Issue by chrisjlee** [miragejs/miragejs#770]
Is there a way to mimic long polling with miragejs ? I guess the model will just change after an interval?

**Comment by samselikoff** [comment]
Yes exactly, I’ve done something like this before. setInterval in seeds or somewhere else and cal server.create inside 👍
